src/drivers/boards/px4fmu-v2/px4fmu_init.c

```diff
@@ -185,6 +185,13 @@ __EXPORT void board_on_reset(int status)
 	 */
 
 	if (status >= 0) {
+		/* on reboot (status >= 0) power off sensors and peripherals */
+		stm32_configgpio(GPIO_VDD_3V3_SENSORS_EN);
+		stm32_gpiowrite(GPIO_VDD_3V3_SENSORS_EN, 0);
+
+		stm32_configgpio(GPIO_VDD_5V_PERIPH_DIS);
+		stm32_gpiowrite(GPIO_VDD_5V_PERIPH_DIS, 1);
+
 		up_mdelay(400);
```

Comment:
Increasing this might unstuck the sensors.

Reply:
No luck so far.